S02-types/int-uint.t

```diff
@@ -59,25 +57,17 @@ for @inttypes -> $type {
           "$type underflows to $maxval";
     }
 
-    if $type eq "uint64" {
+    # XXX TODO: merge this if/else into one test once the fudge isn't needed
+    if $type eq "uint64" || $type eq "int64" {
         #?rakudo.jvm todo 'setting to more than max'
         throws-like { EVAL "my $type \$var = {$maxval+1}" },
           Exception,
           "setting $type to more than $maxval throws";
     } else {
-        # XXX TODO: merge this if/else into one test once the fudge isn't needed
-        if $type eq 'int64' {
-            #?rakudo.jvm todo 'setting more than max throws'
-            throws-like { EVAL "my $type \$var = {$maxval+1}" },
-              Exception,
-              "setting $type to more than $maxval throws";
-        }
-        else {
-            #?rakudo todo 'setting more than max throws'
-            throws-like { EVAL "my $type \$var = {$maxval+1}" },
-              Exception,
-              "setting $type to more than $maxval throws";
-        }
+        #?rakudo todo 'setting more than max throws'
```

Comment:
Don't these pass for the MoarVM backend now?

Reply:
I don't think so, for example:
```
$ ./rakudo-m -e 'my int8 $var = 128; dd $var'
-128
```

Only for the ```int64``` and ```uint64``` cases there is an exception:
```
$ ./rakudo-m -e 'my $maxval = (::int64).Range.int-bounds[1]; say $maxval; my int64 $var = $maxval + 1'
9223372036854775807
Cannot unbox 64 bit wide bigint into native integer
  in block <unit> at -e line 1
$ ./rakudo-m -e 'my $maxval = (::uint64).Range.int-bounds[1]; say $maxval; my uint64 $var = $maxval + 1'
18446744073709551615
Cannot unbox 65 bit wide bigint into native integer
  in block <unit> at -e line 1
```